client/src/core/format.js

```diff
@@ -41,7 +41,8 @@ const compact = (precision, val, lang, options) => {
   let symbol;
   let new_val;
   if (val === 0){ 
-    return "0";
+    new_val = 0;
+    symbol = "";
```

Comment:
Is this okay?

Reply:
Probably? It looks like that could generate a bug in cases where `raw` wasn't set.